src/app_service/service/token/dto.nim

```diff
@@ -54,6 +57,8 @@ proc toTokenDto*(jsonObj: JsonNode, isVisible: bool, hasIcon: bool = false, isCu
   discard jsonObj.getProp("symbol", result.symbol)
   discard jsonObj.getProp("decimals", result.decimals)
   discard jsonObj.getProp("color", result.color)
+  if jsonObj.hasKey("communityId"):
+    result.communityId = jsonObj["communityId"].getStr
```

Comment:
you can jsonObj{"communityId"}.getStr to check for nil or key not present in one line i believe